Assets/Resources.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
```

Comment:
Is it an accident that this .meta file deletion is here?

Reply:
I don't know why Unity has a problem here. It's still in the first line of `fileFormatVersion: 2`, maybe there were two such lines.